CHANGES.txt

```diff
@@ -5,6 +5,12 @@ Changes for crate
 Unreleased
 ==========
 
+- Improved SQLAlchemy's ``CrateDialect.get_pk_constraint`` to be compatible
+  with breaking changes in CrateDB >=5.1.0. Since CrateDB 3.0.0 already, it
+  would have been correct to use the ``information_schema.key_column_usage.table_schema``
+  column instead of ``table_catalog`` (mostly contains value ``doc``, the
+  default schema of CrateDB). Now, with CrateDB >=5.1.0, ``table_catalog`` will
```

Comment:
Not sure about details - is it important for users? First sentence looks sufficient to me.